aws-auth/src/main/java/software/amazon/awssdk/awsauth/credentials/internal/ConnectionUtils.java

```diff
@@ -13,7 +13,7 @@
  * permissions and limitations under the License.
  */
 
-package software.amazon.awssdk.core.internal.net;
+package software.amazon.awssdk.awsauth.credentials.internal;
```

Comment:
For ConnectionUtils we should consider refactoring to use the SDK HTTP client instead of URL connection. Also we should consider putting the ec2 metadata "client" into it's own module.